From 682a699e71147673f8de5fa12dbec8f2a4c28d9c Mon Sep 17 00:00:00 2001
From: David Herrmann <dh.herrmann@gmail.com>
Date: Thu, 12 Mar 2015 12:46:46 +0100
Subject: [PATCH] man: split paragraph

Explicitly put the "multiple EnvironmentFile=" description into its own
paragraph to make it much easier to find.

(cherry picked from commit f407824d751a9cb31abfdf0343fe179e0efef259)
---
 man/systemd.exec.xml | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/man/systemd.exec.xml b/man/systemd.exec.xml
index 11b160e58..fdb157864 100644
--- a/man/systemd.exec.xml
+++ b/man/systemd.exec.xml
@@ -283,7 +283,9 @@
         shortly before the process is executed (more specifically,
         after all processes from a previous unit state terminated.
         This means you can generate these files in one unit state, and
-        read it with this option in the next). Settings from these
+        read it with this option in the next).</para>
+
+        <para>Settings from these
         files override settings made with
         <varname>Environment=</varname>. If the same variable is set
         twice from these files, the files will be read in the order
